packages/@aws-cdk/aws-s3-deployment/README.md

```diff
@@ -44,6 +44,12 @@ This is what happens under the hood:
    `websiteBucket`). If there is more than one source, the sources will be
    downloaded and merged pre-deployment at this step.
 
+**IMPORTANT** The `aws-s3-deployment` module is only intended to be used with
+assets built by the CDK CLI and Zip files from sources you trust. If you use
+`Source.bucket()` to reference a zip file from an untrusted source, it might
```

Comment:
I think placing this under "Supported sources" makes slightly more sense since `Source.bucket` hasn't been introduced yet (though it does make this important message slightly harder to find)

Reply:
Yes. I guess I will remove the reference to `Source.bucket()` here. I had originally added it to allay fears, make it clear to people that if they were using `Source.asset()` they didn't have to worry. But good point on the order of reading.